internal/iam/command_service_account_use.go

```diff
@@ -0,0 +1,43 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/resource"
+)
+
+func (c *serviceAccountCommand) newUseCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "use <id>",
+		Short:             "Choose a service account to be used in subsequent commands.",
+		Long:              "Choose a service account to be used in subsequent commands which support passing a service account with the `--service-account` flag.",
```

Comment:
There are a lot of commands that allow the `--service-account` flag. Have you validated that this works with commands other than the Flink commands?

Reply:
Yeah I did this at first, specifically for api-key `create` and `list`, but then I ran into the problem again that you can't `unset` a key and since service-account is an optional flag for those commands, I decided to remove it again. What do you think, should I still make it work with those commands?

Reply:
The long description seems to imply that `confluent iam service-account use` can be used in conjunction with any command that has `--service-account`, so let's either 1) support that or 2) remove this command.

My vote is for #2 based on:
> the problem again that you can't unset a key

Reply:
Alright, I'll just move it under `flink`, that way it won't affect the other commands but we can still use it for Flink specific ones.

Reply:
I've added it now on the other commands as well. I also implemented a `unset` command for service accounts.